### PR TITLE
Update Gardener docu

### PR DESCRIPTION
**Pull request by vasu1124**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [X] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by vasu1124** [comment]
@kfaseela anything else needed from me?

File: content/en/docs/setup/platform-setup/gardener/index.md [hunk 1]
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
-This page was last updated June 28, 2019.
+## Bootstrapping Gardener
 
-{{< boilerplate untested-document >}}
+To set up your own [Gardener](https://gardener.cloud) for your organization's Kubernetes-as-a-Service needs, follow the
+[documentation](https://github.com/gardener/gardener/blob/master/docs/README.md).
+For testing purposes, you can set up [Gardener on your laptop](https://github.com/gardener/gardener/blob/master/docs/development/getting_started_locally.md) by checking out the source code repo and simply running `make kind-up gardener-up` (the easiest developer way of checking out Gardener!).
```
Comment (kfaseela) [inline content/en/docs/setup/platform-setup/gardener/index.md:17]: source code repository

File: content/en/docs/setup/platform-setup/gardener/index.md [hunk 2]
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
...
+[documentation](https://github.com/gardener/gardener/blob/master/docs/README.md).
+For testing purposes, you can set up [Gardener on your laptop](https://github.com/gardener/gardener/blob/master/docs/development/getting_started_locally.md) by checking out the source code repository and simply running `make kind-up gardener-up` (the easiest developer way of checking out Gardener!).
 
-## Bootstrapping Gardener
+Alternatively, [`23 Technologies GmbH`](https://23technologies.cloud/) offers a fully-managed Gardener service that conveniently works with all supported cloud providers and comes with a free trial: [`Okeanos`](https://okeanos.dev/). Similarly, cloud providers such as [`STACKIT`](https://stackit.de/), [`B'Nerd`](https://bnerd.com/), [`MetalStack`](https://metalstack.cloud/), and many others run Gardener as their Kubernetes Engine.
+
+To learn more about the inception of this open source project, read [Gardener Project Update](https://kubernetes.io/blog/2019/12/02/gardener-project-update/), [Gardener - The Kubernetes Botanist](https://kubernetes.io/blog/2018/05/17/gardener/) on [`kubernetes.io`](https://kubernetes.io/blog).
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/gardener/index.md:21]:
```suggestion
To learn more about the inception of this open source project, read [Gardener Project Update](https://kubernetes.io/blog/2019/12/02/gardener-project-update/) and [Gardener - The Kubernetes Botanist](https://kubernetes.io/blog/2018/05/17/gardener/) on [`kubernetes.io`](https://kubernetes.io/blog).
```

File: content/en/docs/setup/platform-setup/gardener/index.md [hunk 3]
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
...
 
-1.  [Configure access to your Gardener project](https://kubernetes.io/docs/tasks/tools/install-kubectl/#verifying-kubectl-configuration)
+1.  [Configure access to your Gardener project](https://gardener.cloud/docs/dashboard/usage/gardener-api/)
     using a kubeconfig. If you are not the Gardener Administrator already, you
     can create a technical user in the Gardener dashboard: go to the "Members"
     section and add a service account. You can then download the kubeconfig for
-    your project. You can skip this step if you create your cluster using the
-    user interface; it is only needed for programmatic access, make sure you set
-    `export KUBECONFIG=garden-my-project.yaml` in your shell.
+    your project. Make sure you set `export KUBECONFIG=garden-my-project.yaml` in your shell.
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/gardener/index.md:43]: You might replace set with add, or maybe just remove it altogether.
```suggestion
    your project. Make sure you `export KUBECONFIG=garden-my-project.yaml` in your shell.
```

File: content/en/docs/setup/platform-setup/gardener/index.md [hunk 4]
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
...
     ![Download kubeconfig for Gardener](images/gardener_service_account.png "downloading the kubeconfig using a service account")
+    You can skip this step if you intend to create your cluster using the
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/gardener/index.md:45]: Does it make sense to move the statement of optionally doing this step to the front of the step?

File: content/en/docs/setup/platform-setup/gardener/index.md [hunk 5]
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
...
+    user interface; it is only needed for programmatic access.
 
 ### Creating a Kubernetes cluster
 
 You can create your cluster using `kubectl` cli by providing a cluster
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/gardener/index.md:50]:
```suggestion
You can create your cluster using the `kubectl` cli by providing a cluster
```

File: content/en/docs/setup/platform-setup/gardener/index.md [hunk 6]
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
...
 specification yaml file. You can find an example for GCP
 [here](https://github.com/gardener/gardener/blob/master/example/90-shoot.yaml).
 Make sure the namespace matches that of your project. Then just apply the
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/gardener/index.md:53]:
```suggestion
Make sure the namespace matches that of your project. Then apply the
```

File: content/en/docs/setup/platform-setup/gardener/index.md [hunk 7]
```diff
@@ -10,56 +10,56 @@ owner: istio/wg-environments-maintainers
...
-prepared so-called "shoot" cluster CRD with `kubectl`:
+prepared so-called "shoot" cluster manifest with `kubectl`:
 
 {{< text bash >}}
 $ kubectl apply --filename my-cluster.yaml
 {{< /text >}}
 
 The easier alternative is to create the cluster following the cluster creation
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/gardener/index.md:60]:
```suggestion
An easier alternative is to create the cluster following the cluster creation
```

File: content/en/docs/setup/platform-setup/gardener/index.md [hunk 8]
```diff
@@ -70,8 +70,8 @@ Gardener dashboard or via cli as follows:
 $ kubectl --namespace shoot--my-project--my-cluster get secret kubecfg --output jsonpath={.data.kubeconfig} | base64 --decode > my-cluster.yaml
 {{< /text >}}
 
-This kubeconfig file has full administrators access to you cluster. For the rest
-of this guide be sure you have `export KUBECONFIG=my-cluster.yaml` set.
+This kubeconfig file has full administrator access to you cluster.
+For any activities with the payload cluster be sure you have `export KUBECONFIG=my-cluster.yaml` set.
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/gardener/index.md:74]: Not sure if you need the set. Maybe this (and similar in the prior comment) should be ... exported `KUBECONFIG=my-cluster.yaml`?

File: content/en/docs/setup/platform-setup/gardener/index.md (hunk 4 above)
Comment (vasu1124) [reply content/en/docs/setup/platform-setup/gardener/index.md]: thank you. embedded that statement in a <tip/>

File: content/en/docs/setup/platform-setup/gardener/index.md (hunk 8 above)
Comment (vasu1124) [reply content/en/docs/setup/platform-setup/gardener/index.md:78]: I will leave as is for c&p.

**Comment by vasu1124** [comment]
@ericvn thanks for the review. incorporated the feedback.
@kfaseela cleaned up outdated images, squashed, and final lint completed. ready from my side.

**Comment by kfaseela** [comment]
LGTM.. Leaving to Eric for merging

**Review by ericvn — APPROVED** [review]

**Pull request merged**